app/assets/javascripts/discourse/models/login-method.js.es6

```diff
@@ -90,15 +90,6 @@ export function findAll(siteSettings, capabilities, isMobileDevice) {
     methods.pushObject(LoginMethod.create(provider));
   });
 
-  // On Mobile, Android or iOS always go with full screen
```

Comment:
yeah we still need to make this decision somewhere, but it can be in the click handler instead.

Reply:
good catch, let me verify that we don't change existing behaviour here.

Reply:
... and done!